src/MySql/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.3.0
```

Comment:
We need to VersionController to bump the version.

Reply:
Done